pkg/constants/constants.go

```diff
@@ -35,30 +35,30 @@ const (
 	SidecarDefaultMemoryLimit = "128M"
 	PVCStorageSize            = "1Gi"
 
-	// WorkspaceIDLabel is label key to store workspace identifier
-	WorkspaceIDLabel = "controller.devfile.io/workspace_id"
+	// DevWorkspaceIDLabel is label key to store workspace identifier
+	DevWorkspaceIDLabel = "controller.devfile.io/devworkspace_id"
+
+	// DevWorkspaceNameLabel is label key to store workspace name
+	DevWorkspaceNameLabel = "controller.devfile.io/devworkspace_name"
```

Comment:
only `name` : `controller.devfile.io/name` ?

Reply:
but this label is propagate to all devworkspace related stuff, pod, service, route.... It may be not clear enough to see `controller.devfile.io/name` label.

Reply:
duplicated